i18n/en.pot

```diff
@@ -100,13 +103,16 @@ msgstr ""
 msgid "Search for items to add to this dashboard"
 msgstr ""
 
-msgid "See"
+msgid "See fewer {{itemTypes}}"
+msgstr ""
+
+msgid "See more {{itemTypes}}"
 msgstr ""
 
-msgid "fewer"
+msgid "+ ADD"
```

Comment:
Should the + and the uppercase format be part of the translation? Or can these be handled outside of i18n?

Reply:
Yes, I thought about that as well. I'll update it.